web_apply_field_style/models/base.py

```diff
@@ -0,0 +1,28 @@
+from odoo import api, models
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
+
+    @api.model
+    def _get_view(self, view_id=None, view_type="form", **options):
+        arch, view = super()._get_view(view_id, view_type, **options)
+        if view_type == "form":
+            self._update_css_class(arch)
+        return arch, view
+
+    def _update_css_class(self, arch):
+        css = self._get_field_styles()
+        if css.get(self._name):
+            for style in css[self._name]:
```

Comment:
```suggestion
            for style in css.get(self._name, []):
```

Reply:
+1, and deserves a specific test (generate a form view of a model with no style and check the view does not crash).

Useful module by the way, thank you for sharing!

Reply:
Yes, finally added with extra tests for different invalids entries. Thanks to you both